metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
I'm OK with the current config approach. But I haven't tried adding additional counters yet. Let's get some people using it and see what kind of feedback we get.